src/components/GridTable/Body.tsx

```diff
@@ -0,0 +1,13 @@
+import { Box, PropsWithBox } from "@saleor/macaw-ui-next";
+import React from "react";
+
+type GridTableBodyElement = React.ElementRef<"tbody">;
+
+export const GridTableBody = React.forwardRef<GridTableBodyElement, PropsWithBox<unknown>>(
```

Comment:
question: have you tries to type this `unknow` like this `JSX.IntrinsicElements["tbody"]`?

Reply:
Fixed